module/LocalStorage.js

```diff
@@ -0,0 +1,30 @@
+class LocalStore {
+  static idGenerator() {
```

Comment:
- [ ]  Please it would be better if make this method as `Arrow functions` instead of `normal functions` that will make use the Es6 features

Reply:
Thanks so much for pointing that out. I must have missed it. I appreciate. Cheers 👍🏾